packages/authentication/modules/access/useResetPassword/__tests__/useResetPassword.test.tsx

```
@@ -1,31 +1,21 @@
-  mockFetchError,
-  renderHook,
-} from '@baseapp-frontend/test'
+import { ComponentWithProviders, MockAdapter, renderHook } from '@baseapp-frontend/test'
+import { axios } from '@baseapp-frontend/utils'
 
 import { z } from 'zod'
 
 import useResetPassword from '../index'
 
+// @ts-ignore TODO: (BA-1081) investigate AxiosRequestHeaders error
+export const axiosMock = new MockAdapter(axios)
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)